devops/pipeline/stages/e2etest_baremetal.yaml

```diff
@@ -135,4 +136,5 @@ stages:
       deviceStageName: Baremetal_${{ parameters.deviceId }}
       testExpression: 'TestCategory != BaremetalExclude'
       twinTimeout: ${{ parameters.twinTimeout }}
-      environment: ${{ parameters.environment }}
\ No newline at end of file
+      environment: ${{ parameters.environment }}
+      distroName: debian9
```

Comment:
The hardcoded `debian9` is fine for now, we are changing all our targets to be generic going forward, no more cloud/baremetal